src/main/UseCases/AddDS.java

```diff
@@ -0,0 +1,22 @@
+package UseCases;
+
+public class AddDS {
```

Comment:
rigorously followed clean architecture, this DS is an input data structure, and should be used by AddUseCase.